// @test error

fn main() : int
{
	while (3)
	{
		print_uint32(0);
	}

	return 1;
}